app/models/asset_tag_import.rb

```diff
@@ -114,4 +114,10 @@ def apply
       end
     end
   end
+
+  # part of the errors interface
+  # not bothering with localization
+  def self.human_attribute_name(attr_name, *_)
```

Comment:
are these hacks?

Reply:
Yea.

`full_messages` (part of `ActiveModel#Errors`) asks the model for the name of the attributes responsible for the error. The internationalization comes from this method.

This is the only method on `ActiveModel#errors` that has been part of the interface for a while and is nailed down in the interface. All other ways of looking at errors seems to have been changing over the versions

Reply:
I guess my concern is that this will break the UI dictionary that we have that leverage the Ruby i18n (for better or worse).  I'm also curious why we need this at all...I would expect this to be convention over configuration and not needed at all.

Reply:
This is an import class, not a model class.
Typically we just use the implementation from `ActiveRecord::Base`.
If you look at the [docs](https://api.rubyonrails.org/v6.1.3/classes/ActiveModel/Errors.html) their example for this method is basically the one I implemented.

Since we are not calling `full_messages` in the ui, localization doesn't matter here. I can research adding messages if we need but it would only be for some future edge case that I am not sure we would use.

Reply:
> This is an import class, not a model class.

Oh i see now, it's doesn't derive from ActiveRecord::Base.  These arguably should be removed from the models directory entirely and put into the lib directory, but obviously not for this PR.